common/src/chain/config/builder.rs

```diff
@@ -55,7 +56,23 @@ impl ChainType {
                 ];
                 NetUpgrades::initialize(upgrades).expect("net upgrades")
             }
-            ChainType::Testnet => unimplemented!("Testnet upgrades"),
+            ChainType::Testnet => {
+                let pos_config = crate::chain::create_unittest_pos_config();
+                let upgrades = vec![
+                    (
+                        BlockHeight::new(0),
+                        UpgradeVersion::ConsensusUpgrade(ConsensusUpgrade::IgnoreConsensus),
+                    ),
+                    (
+                        BlockHeight::new(1),
+                        UpgradeVersion::ConsensusUpgrade(ConsensusUpgrade::PoS {
+                            initial_difficulty: crate::chain::pow::limit(ChainType::Testnet).into(),
```

Comment:
I just noticed that this is `pow`. Should we instead set this in `crate::chain::pos` too so that PoS isn't relying on something defined in PoW code?

Reply:
There should be a separate limit for pos, granted.

Reply:
Should be fixed now.